content/updates/2016-12-20-ew-sandbox-deprecation.md

```diff
@@ -0,0 +1,19 @@
++++
+date = "2016-12-20"
+title = "East/West Sandbox Deprecation Notice"
++++
+
+The original cloud.gov in AWS East/West is now officially deprecated and will be retired soon. We will be in touch to assist with migration of Prototyping, FISMA Moderate, and FISMA Low organizations to the new environment shortly. 
+
+Sandbox organizations will be retired as of December 31st. If you have hosted anything in a sandbox space that you would like to preserve, you must back it up or [migrate it to the new environment]({{< relref "docs/apps/govcloud.md" >}}) yourself.
```

Comment:
I would word this `Sandbox organizations in East/West will be retired as of December 31st.` just to be especially clear that it only applies to E/W.